app/src/main/java/com/kickstarter/viewmodels/CreatorDashboardViewModel.java

```diff
@@ -0,0 +1,107 @@
+package com.kickstarter.viewmodels;
+
+import android.support.annotation.NonNull;
+import android.util.Pair;
+
+import com.kickstarter.libs.ActivityViewModel;
+import com.kickstarter.libs.Environment;
+import com.kickstarter.libs.RefTag;
+import com.kickstarter.libs.rx.transformers.Transformers;
+import com.kickstarter.libs.utils.ListUtils;
+import com.kickstarter.libs.utils.NumberUtils;
+import com.kickstarter.libs.utils.ProjectUtils;
+import com.kickstarter.models.Project;
+import com.kickstarter.services.ApiClientType;
+import com.kickstarter.services.apiresponses.ProjectsEnvelope;
+import com.kickstarter.ui.activities.CreatorDashboardActivity;
+
+import java.util.List;
+
+import rx.Notification;
+import rx.Observable;
+import rx.subjects.PublishSubject;
+
+public interface CreatorDashboardViewModel {
+  interface Inputs {
+    void projectViewClicked();
+  }
+
+  interface Outputs {
+
```

Comment:
lil extra newline, also would be nice to add some `/** Coments */` to these Inputs and Outputs!